doc/api/child_process.md

```diff
@@ -281,9 +284,11 @@ changes:
 The `child_process.execFile()` function is similar to [`child_process.exec()`][]
 except that it does not spawn a shell by default. Rather, the specified
 executable `file` is spawned directly as a new process making it slightly more
-efficient than [`child_process.exec()`][].
+efficient than [`child_process.exec()`][]. Also `child_process.execFile()` takes
+an array of strings as an argument while the other takes a single command
+string.
```

Comment:
I'm not sure we need to put this information into prose like this. It's in the function signature.

Reply:
I'm not sure this is the best way to express it, but I do think we have to say something about how exec and execFile differ. If we had to do this again, both functions would *not* exist, there would be only `exec()`, since they now differ only in two things: the default for options.shell, the cmd vs arvgv signature. I think its worth calling this out, because its confusing, and so that people don't have to do a line by line comparison of the two functions to see that those are the only two differences - we can just say it straight up. It probably is not possible to refactor this here, but this is one reason I dislike cut-n-paste in docs as much as in code: if two functions link to the same core docs, its instantly clear that they share the same core behaviour, with cut-n-paste, it takes very careful reading. From a historical perspective, the exec and execFile docs weren't literally copied from each other, its just they got mutual feature envy, and eventually every feature one had was also added to the other so it appears as if the docs were copied.  Its possible that the child_process docs could become much more readable by compressing the multiple copies of the arg docs into core sections, linked to by the APIs that expose that behaviour, but its also possible it would make them more confusing.